README.md

```diff
@@ -24,3 +24,28 @@ You can find an example flow that shows how to use the insert node in `examples/
 ![Clarify Insert Node](https://github.com/searis/node-red-contrib-clarify/blob/master/examples/random-data-example.png?raw=true)
 
 Any questions? Send us an email on support@clarify.us
+
+## Changelog
+
+From 1.0.0 we will not do breaking changes to the overall msg.format. 
+
+### 1.0.0 - Breaking changes from -beta.X
```

Comment:
Nice change log.

But should we do another beta?

I don't think we want to do any more changes to the input structure, but I don't think we have yet nailed the output yet.

Reply:
#37